### PR TITLE
docs: update generate arm message

### PR DESCRIPTION
**Pull request by wenytang-ms**
https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/12624011

### CODE REVIEW

File: packages/fx-core/resource/strings.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "AlreadyProvisionNotice": "'%s' is already provisioned in the cloud, will create the teams app again if it has been removed.",
     "NotProvisionedNotice": "'%s' is not provisioned in the cloud yet. Please provision first.",
     "GenerateArmTemplateFailNotice": "'%s' Failed to create ARM template for required Azure services.",
-    "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. The existing template file will be moved to backup folder under Azure resource template folder for your reference. We recommend to initialize your project workspace with git init to track all changes.",
+    "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. We recommend to initialize your project workspace with git init to track all changes.",
```
Comment (MuyangAmigo) [inline packages/fx-core/resource/strings.json:26]: Maybe change to: `The Azure resource template is updated based on your resource and capability selection.....`

Comment (wenytang-ms) [reply packages/fx-core/resource/strings.json:26]: sure, thanks!

File: packages/fx-core/resource/strings.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "AlreadyProvisionNotice": "'%s' is already provisioned in the cloud, will create the teams app again if it has been removed.",
     "NotProvisionedNotice": "'%s' is not provisioned in the cloud yet. Please provision first.",
     "GenerateArmTemplateFailNotice": "'%s' Failed to create ARM template for required Azure services.",
-    "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. The existing template file will be moved to backup folder under Azure resource template folder for your reference. We recommend to initialize your project workspace with git init to track all changes.",
+    "RegenerateArmTemplateConfirmNotice": "The Azure resource template is updated based on your resource and capability selection. We recommend to initialize your project workspace with git init to track all changes.",
```
Comment (blackchoey) [inline packages/fx-core/resource/strings.json:26]: Should we change the key of this message?

File: packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts [hunk 3]
```diff
@@ -326,14 +323,9 @@ export async function addCapability(
   return ok({});
 }
 
-export async function confirmRegenerateArmTemplate(ui?: UserInteraction): Promise<boolean> {
+export function confirmRegenerateArmTemplate(ui?: UserInteraction) {
```
Comment (blackchoey) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/executeUserTask.ts:326]: also update the function name?

**Review by MuyangAmigo — APPROVED** [review]
Approve for String change.

File: packages/fx-core/tests/plugins/solution/solution.usertask.test.ts [hunk 4]
```diff
@@ -283,7 +283,8 @@ describe("V2 implementation", () => {
     mockScaffoldCodeThatAlwaysSucceeds(appStudioPluginV2);
     mockScaffoldCodeThatAlwaysSucceeds(localDebugPluginV2);
     mockScaffoldCodeThatAlwaysSucceeds(frontendPluginV2);
-
+    const insiderPreviewFlag = process.env.TEAMSFX_INSIDER_PREVIEW;
```
Comment (wenytang-ms) [inline packages/fx-core/tests/plugins/solution/solution.usertask.test.ts:286]: @1yefuwang1 seems there need to sync up with you, on "v2/executeUserTask" generateArmTemplate need to support selectedPlugins parameters

**Review by jayzhang — APPROVED** [review]

**Pull request merged**